includes/shortcodes/class.llms.shortcodes.blocks.php

```diff
@@ -63,7 +63,9 @@ public function __construct() {
 	 * @return void
 	 */
 	public function register_blocks(): void {
-		foreach ( $this->shortcodes as $shortcode ) {
+		$shortcodes = array_keys( $this->shortcodes );
```

Comment:
Shall we filter the `$this->shortcodes` here?
so to maybe allow 3rd parties (or our add-ons) to hook into this?

Reply:
Yes, much better idea

Reply:
Added here https://github.com/gocodebox/lifterlms/pull/2411/commits/0f71035050f029afa0f6ca0ac67ee85df7c9ff22, let me know if it's ok to use `fn()` or if you can think of a cleaner way:

```php
'access-plan-button' => array(
    'render' => fn( $atts, $content ) => LLMS_Shortcodes::access_plan_button( $atts, $content ),
),
```

Reply:
I like this, but I'm not convinced.
it's not a problem of `fn` we could always use a closure/anonymous function like `function( $atts, $content )`, it's more that I'm not sure we want all these functions there.
Alternatives? Something like
```
'access-plan-button' => array(
	'render' => array( 'LLMS_Shortcodes', 'access_plan_button' ),
)
```
then handle the thing with `call_user_func()`? would that work?